.github/workflows/mql-mimic-tests.yml

```diff
@@ -0,0 +1,43 @@
+name: MQL Mimic Tests
+
+on:
+  push:
+    branches: [ "**" ]
+
+concurrency:
+  # For pull_request_target workflows we want to use head_ref -- the branch triggering the workflow. Otherwise,
+  # use ref, which is the branch for a push event.
+  group: ${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  trigger-via-zapier:
+    name: Trigger Test Run
+    runs-on: ubuntu-20.04
+    permissions:
+      checks: write
+
+    steps:
+
+      - name: "Trigger MQL Mimic Tests"
+        env:
+          trigger_url: '${{ secrets.MQL_MOCK_TRIGGER }}'
+          branch: '${{ github.ref_name }}'
+          repo: '${{ github.repository }}'
+          token: '${{ secrets.GITHUB_TOKEN }}'
+          sha: '${{ github.sha }}'
+        run: |
+          echo '{"branch":"'$branch'","repo":"'$repo'","token":"'$token'","sha":"'$sha'"}'
```

Comment:
i'm pretty sure these logs aren't publicly accessible, and in either case `GITHUB_TOKEN` gets obfuscated in the logs, right?

Reply:
It does get obfuscated but this was just for debugging and can be removed!